tools/start_all_tmux.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is used to start all of the microservices
+# for the Lucida project by creating a tmux window for each
+# server in the background.
+# Example calls:
+# $./start_all_tmux.sh secure
+# $./start_all_tmux.sh
+# To attach to the tmux session
+# use the following command:
+# $tmux a -t lucida
+
+SESSION_NAME="lucida"
+
+# Check to see if we should run on http/ws (non-secure) or https/wss (secure)
+if [ "$1" == "secure" ]; then
+    echo "Enabling secure host"
+    # Getting the host IP address
+    export ASR_ADDR_PORT="wss://$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}'):8081"
```

Comment:
The issue is we need a common place to have a service registry that all servers and cmd center can access. In some other implementations the command center holds the registry, the microservices register themselves giving the cmd their host and port, and the cmd adds this to its service registry. This way the microservices can change their locations and just register themselves every time they are initialized.